storybook/src/elements/nuxeo-search-results-layout/nuxeo-search-results-layout.stories.js

```diff
@@ -0,0 +1,37 @@
+import { storiesOf } from '@storybook/polymer';
+import { html } from 'lit-html';
+import { LayoutBehavior } from '@nuxeo/nuxeo-ui-elements/nuxeo-layout-behavior.js';
+import { Polymer } from '@polymer/polymer/lib/legacy/polymer-fn.js';
+import '@nuxeo/nuxeo-ui-elements/search/nuxeo-search-results-layout.js';
+import '../../../.storybook/i18n';
+import { codePanelTemplate } from '../code-panel-template.js';
+
+window.Polymer = Polymer;
```

Comment:
perhaps we should create a ticket to move all the compat stuff here https://github.com/nuxeo/nuxeo-web-ui/blob/master/index.js#L4  to a helper import in elements 🤔

Reply:
I don't understand what's the problem here. What's "all the compat stuff"?